user_manual/talk/advanced_features.rst

```diff
@@ -114,41 +114,44 @@ specific room.
+- Start and stop recordings during a call.
+- Record the video and audio stream of the speaker, as well as screen share.
 - Access, share and download recorded files for future reference or distribution.
 
+Enabling this feature requires the recording server to be set up by the system administration.
+
 Manage a recording
-^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^
 
-The moderator of the conversation can start a recording together with a call start or anytime during it. To enable:
+The moderator of the conversation can start a recording together with a call start or anytime during a call:
 
-- **Before the call**: tick a checkbox "Start recording immediately with the call" in Media settings, then click on "Start call".
+- **Before the call**: tick the checkbox "Start recording immediately with the call" in "Media settings", then click on "Start call".
 - **During the call**: click on the top-bar menu, then click "Start recording".
 
 .. image:: images/start-recording-before-call.png
 .. image:: images/start-recording-in-call.png
 
-The recording will start shortly, and you will see a red indicator next to the call time. By clicking on it, you can stop the recording any time.
+The recording will start shortly, and you will see a red indicator next to the call time. You can stop the recording at any time while the call is still ongoing by clicking on that indicator and selecting "Stop recording", or by using the same action in the top-bar menu. If you do not manually stop the recording, it will end automatically when the call ends.
 
 .. image:: images/stop-recording.png
 
-After stopping a recording, the server will take some time to prepare and save a recorded file.
-Moderator, who started the recording, should receive a notification about status of the file. From there it can be shared then to the conversation.
+After stopping a recording, the server will take some time to prepare and save the recorded file. The moderator, who started the recording, receives a notification when the file is uploaded. From there, it can be shared in the chat.
 
+.. image:: images/share-recording-notification.png
 .. image:: images/shared-recordings.png
 
 Recording consent
-^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^
 
-For compliance reasons with various privacy rights it is possible to ask participants for a consent to be recorded before joining the call.
-System administrator can force it for all conversations, or keep it configurable by moderators. Then moderator can see this option in Conversation settings:
+For compliance reasons with various privacy rights, it is possible to ask participants for consent to be recorded before joining the call.
+The system administration can enable mandatory consent for all conversations system-wide or allow moderators to configure this option on a conversation level. If your system is set up accordingly, moderators can access the following option in the conversation settings:
```

Comment:
Specify which set up you are referring to in the second sentence, like :  
`If your system setup of recording consent is moderator-configurable, moderators can access the following option in the conversation settings:`